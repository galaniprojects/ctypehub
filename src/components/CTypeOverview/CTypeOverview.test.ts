import { describe, expect, it } from '@jest/globals';

import { getSnapshotHtmlForPath } from '../../../testing/getSnapshotHtmlForPath';

describe('CTypeOverview', () => {
  it('should match snapshot', async () => {
    const html = await getSnapshotHtmlForPath('test/CTypeOverview-example');
    expect(html).toMatchSnapshot();
  });
});
