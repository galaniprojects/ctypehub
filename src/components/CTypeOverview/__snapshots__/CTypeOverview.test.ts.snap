// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CTypeOverview > should match snapshot 1`] = `
<li class="_container container">
  <h2><a href="/ctype/kilt:ctype:0x1" class="title">Example CType</a></h2>
  <p class="description">This is some example cType data</p>
  <ul class="tags">
    <li class="tag"><a href="/tag/example">#example</a></li>
  </ul>
  <p class="attestations"><span>Number of Attestations:</span><span>1</span></p>
</li>
`;
